framework/kms-keyring.md

```diff
@@ -74,15 +74,15 @@ See [AWS Documentation](https://docs.aws.amazon.com/kms/latest/APIReference/API_
 
 An AWS KMS API for decrypting ciphertext previously encrypted by [GenerateDataKey](#aws-kms-generatedatakey) or [Encrypt](#aws-kms-encrypt).
 
-See [AWS Documenetation](https://docs.aws.amazon.com/kms/latest/APIReference/API_Decrypt.html).
+See [AWS Documentation](https://docs.aws.amazon.com/kms/latest/APIReference/API_Decrypt.html).
 
 ## Inputs
 
 On keyring initialization, a keyring MUST define the following:
 
 - [Client Supplier](#client-supplier)
 
-On keyring initialization, a keyring MAY define the following:
+On keyring initialization, a keyring MUST optionally define the following:
```

Comment:
What about "MUST accept the following as optional inputs"?

In which case I'd also edit the above to say "a keyring MUST accept the following as required inputs:"

Reply:
We could also address this in a separate PR which standardizes across all keyrings, and merge this in as is.

Reply:
Addressing concern in https://github.com/awslabs/aws-encryption-sdk-specification/pull/213